### DISCUSSION TITLE
Can the postalserver set up sender and receiver email avatars?

### DISCUSSION THREAD

**Discussion by wangyuanchen** [postalserver/postal#2962] [Help with using Postal] [answered]
I have a requirement here that I need to set up a avatar for sending and receiving emails, but I don't see this feature in the configuration description. I'm not sure if it's a problem with my usage or if it's currently not possible to set avatar information. May I ask for your advice now？

**Comment by willpower232** [comment, marked as answer]
Avatars aren't part of the message, they are added by the receiving email software using Gravatar (or similar if there are any other options).
